app/views/articles/create.js.erb

```diff
@@ -0,0 +1,7 @@
+$(document).ready ->
```

Comment:
Since this js is getting returned directly to the browser to run on in the context of the existing page, you don't need the document.ready check.